src/main/java/org/worshipsongs/importer/SongParser.java

```diff
@@ -64,4 +88,75 @@ public String findMatchingData(String input, String attributeName)
         }
         return matchingData;
     }
+
+    public String getXmlLyrics(String lyrics, String verseOrder)
+    {
+        this.lyrics = lyrics;
+        this.verseOrder = verseOrder;
+
+        Writer out = new StringWriter();
+        try {
+            DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+            Document document = docBuilder.newDocument();
+
+            getSongTag(document);
+
+            Transformer transformer = TransformerFactory.newInstance().newTransformer();
+            document.setXmlStandalone(true);
+            transformer.transform(new DOMSource(document), new StreamResult(out));
+
+            System.out.print("Data:"+out.toString());
+        } catch (ParserConfigurationException pce) {
+            pce.printStackTrace();
+        } catch (TransformerException tfe) {
+            tfe.printStackTrace();
+        }
+        return out.toString();
+    }
+
+    public void getSongTag(Document document)
```

Comment:
This method is not returning anything. Consider changing the method name to "addSongTag".